packages/babel-types/src/definitions/es2015.js

```diff
@@ -260,6 +261,7 @@ defineType("ExportNamedDeclaration", {
       validate: assertNodeType("StringLiteral"),
       optional: true,
     },
+    exportKind: validateOptional(assertOneOf("type", "value", "let")),
```

Comment:
When is it `let`?

Reply:
found it in astexplorer when trying:
`export let something=1`

but can not reproduce it now… - will remove it

Reply:
Yeah, that should be a `value` export.